mdl/src/main/herd/scripts/configureAndStartHerd.sh

```diff
@@ -88,6 +88,23 @@ function addStackTagsToSqs(){
     fi
 }
 
+function waitIndexToBeReady(){
+    indexName=$1
+    index_status=$(curl --silent --request GET --header 'Accept: application/json' --user ${herdAdminUsername}:${herdAdminPassword} ${httpProtocol}://${herdLoadBalancerDNSName}/herd-app/rest/searchIndexes/${indexName} | jq -r '.searchIndexStatus')
+    echo index_status="${index_status}"
+    while [ "${index_status}" == "BUILDING" ]
+    do
+      sleep 30s
+      index_status=$(curl --silent --request GET --header 'Accept: application/json' --user ${herdAdminUsername}:${herdAdminPassword} ${httpProtocol}://${herdLoadBalancerDNSName}/herd-app/rest/searchIndexes/${indexName} | jq -r '.searchIndexStatus')
+      echo index_status="${index_status}"
+    done
+    if [ "${index_status}" != "READY" ]
+    then
+        error "Invalid search index status. Status: ${index_status}"
```

Comment:
This fails becasuse `error` is not defined as a function/command, probably a copy-paste error.

Reply:
Also, the request needs to be made using the --insecure switch for it to go through. I've put in a fix.

Reply:
1. I already said it's a copy-paste issue, and I found this issue because execution failed. The command produces NO output because the request fails. 
2. Have you tried not using the --insecure option? Does it work when you have auth enabled on the stack? Do you think the domain name of the ACM certificate matches the domain-name of the Load balancer A-name?